src/eval-frame/components/panes/console/console-language-menu.jsx

```diff
@@ -83,7 +83,8 @@ const ConsoleLanguageMenuUnconnected = ({
             <MenuItem
               key={language.languageId}
               onClick={onMenuClickCreator(
-                setConsoleLanguage,
+                languageId =>
+                  sendActionToEditor(setConsoleLanguage(languageId)),
```

Comment:
as above, the substantive change in this file is that this component no longer dispatches, but rather sends a action to the editor to set the console language